src/patternfly/demos/Page/page-demo-expandable-nav.hbs

```diff
@@ -84,11 +84,6 @@
             {{/nav-list}}
           {{/nav-subnav}}
         {{/nav-item}}
-        {{#if page--IsLegacySeparators}}
-          {{#> nav-separator}}{{/nav-separator}}
-        {{else}}
-          {{#> divider divider--type="li"}}{{/divider}}
-        {{/if}}
```

Comment:
Did you want to pull the nav-separator out of this component in this PR? Works for me if you do, I saw this and removed that element from the documentation, but realized we still need to yank out

https://github.com/mattnolting/patternfly/blob/3fa5a9097a7487fc87d84f776ea60218b1ef07d3/src/patternfly/demos/Page/page-demo-expandable-nav.hbs#L45-L49

https://github.com/mattnolting/patternfly/blob/style-vertical-nav/src/patternfly/components/Nav/nav-separator.hbs

and the 2 examples in https://github.com/mattnolting/patternfly/blob/style-vertical-nav/src/patternfly/demos/Page/examples/Page.md that reference `IsLegacySeparators`